core/js/src/score.ts

```diff
@@ -2,6 +2,7 @@ export interface Score {
   name: string;
   score: number | null;
   metadata?: Record<string, unknown>;
+  // Note: this field is deprecated, as errors are propagated up to the caller.
```

Comment:
can you add a DEPRECATION_NOTICE to the comment, so when we grep for it, we can find this message?